gef-ui/package.json

```diff
@@ -34,7 +34,7 @@
     "unit-test-ff": "jest --coverage --verbose --config=unit.ff.jest.config.js --passWithNoTests"
   },
   "dependencies": {
-    "@ministryofjustice/frontend": "1.8.0",
+    "@ministryofjustice/frontend": "3.0.1",
```

Comment:
@ttbarnes can we bump to 3.0.2 as it has my fix for the issue @BethThomas141 was having with the banner class cc @MarRobSoftwire

Reply:
Will do - what was the banner issue?

Reply:
Thewre was an issue with MoJ components not displaying class if you set `type` (see [here](https://github.com/ministryofjustice/moj-frontend/issues/886)). Its since been fixed. If I'm still blocked tomorrow I'll dig out Beth's work around for this and update it.

Reply:
Nice one @AlexBramhill I think we can just bump the MOJ library in this PR instead of the workaround

Reply:
Sounds great yeah!

Reply:
I've bumped MOJ to `3.0.2`. Need to run this branch locally and check some things before it's ready